internal/sirius/person.go

```diff
@@ -36,6 +36,7 @@ type Person struct {
 	CompanyReference      string     `json:"companyReference"`
 	PersonType            string     `json:"personType,omitempty"`
 	Cases                 []*Case    `json:"cases,omitempty"`
+	SystemStatus          *bool      `json:"systemStatus,omitempty"`
```

Comment:
ive used `*bool` to be able to make a distinction between a non-existent boolean and false boolean

Reply:
Do we need to make that distinction? It seems like we treat `false` and `nil` the same